### ISSUE TITLE
Upgrade script  to Activiti 6.0 need ACT_HI_ATTACHMENT add USER_ID_

### ISSUE THREAD

**Issue by sharapov** [Activiti/Activiti#1209]
The script to create the table ACT_HI_ATTACHMENT is a new field USER_ID_.
In the update scripts I can't find it.
alter table ACT_HI_ATTACHMENT add USER_ID_ varchar(255);

**Comment by balsarori** [comment]
From what version are you upgrading to version 6?

**Comment by sharapov** [comment]
5.18
But I found no updates in any of the scripts.

**Comment by balsarori** [comment]
I would recommend you upgrade to the latest 5x version (5.22) then upgrading to version 6

**Comment by sharapov** [comment]
Ok. If I have time I'll try. But I doubt it.
I studied the scripts. There is no adding of this field.
After adding the field, almost everything worked.
Only the start tag c formkey 
<startEvent id="startevent12" name="Start" activiti:initiator="kermit" activiti:formKey="issueFormLite"></startEvent> when deploying
not set HAS_START_FORM_KEY_ field to TRUE.
Maybe I am wrong updated app.
I think to wait what will say others. To not bother You with unnecessary questions. 
Thank you for Your work.

**Comment by balsarori** [comment]
The USER_ID_ is part  ACT_HI_ATTACHMENT table since 5.17
https://github.com/Activiti/Activiti/blob/2473e063fd3cc2ab8a11de11cfbfc2ae3688b762/modules/activiti-engine/src/main/resources/org/activiti/db/create/activiti.hsql.create.history.sql#L115

**Comment by sharapov** [comment]
All right. You pointed out the script create a new table ACT_HI_ATTACHMENT. However, my base did not begin with 5.17 and much earlier. There were a few updates. 5.18 was only on the test. And there was no call for Attachment. But the conversation about that starting with the 5.18 update this field is not included in the script adding a field.
I don't understand how to work the apps which are updated with versions older than 5.17.
Anyway, for my base I solved this problem.
I just pointed out that your upgrade scripts a little wrong.
